src/Vox/Core/CssManager.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @author Marco Troisi <hello@marcotroisi.com>
+ * @created 02.01.15
+ */
+
+namespace Vox\Core;
+
+
+class CssManager implements ResourceInterface
```

Comment:
There is something to manage resources in `moss-twig-bridge` : https://github.com/potfur/moss-twig-bridge